.travis.yml

```diff
@@ -6,7 +6,7 @@ services:
 
 env:
   global:
-    - DOCKER_IMAGE="mellery451/rippled-ci-builder:2019-07-18"
+    - DOCKER_IMAGE="mellery451/rippled-ci-builder:2019-08-19"
```

Comment:
I'd rather get build artifacts from ripple "official" repositories rather than developer specific repositories. No changed needed in this PR, but we should look at changing this in another PR.

Reply:
I agree - it's on my TODO list